sky/data/skystorage.py

```diff
@@ -0,0 +1,386 @@
+"""Skystorage module"""
+import functools
+import os
+import pathlib
+import subprocess
+import time
+from typing import Any, List, Optional
+import urllib.parse
+
+import click
+import psutil
+
+from sky import sky_logging
+from sky.utils import common_utils
+from sky.utils import db_utils
+
+logger = sky_logging.init_logger(__name__)
+
+_CSYNC_BASE_PATH = '~/.skystorage'
+
+_DB = None
+_CURSOR = None
+_CONN = None
+_BOOT_TIME = None
+
+
+def db(func):
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        global _DB, _CURSOR, _CONN, _BOOT_TIME
+
+        def create_table(cursor, conn):
+            cursor.execute("""\
+                CREATE TABLE IF NOT EXISTS running_csync (
+                csync_pid INTEGER PRIMARY KEY,
+                sync_pid INTEGER DEFAULT -1,
```

Comment:
default null?

Reply:
Setting `INTEGER` column's non-exist value to `-1` seems reasonable and widely used :)